examples/dmd/parametric_heat_conduction.py

```diff
@@ -0,0 +1,762 @@
+'''
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+//                       libROM MFEM Example: Parametric_Heat_Conduction (adapted from ex16p.cpp)
+//
+// Compile with: make parametric_heat_conduction
+//
+// =================================================================================
+//
+// In these examples, the radius of the interface between different initial temperatures, the
+// alpha coefficient, and two center location variables are modified.
+//
+// For Parametric DMD (ex. 1) (radius, interpolation):
+//   rm -rf parameters.txt
+//   mpirun -np 8 parametric_heat_conduction -r 0.4 -visit -offline -rdim 16
+//   mpirun -np 8 parametric_heat_conduction -r 0.45 -visit -offline -rdim 16
+//   mpirun -np 8 parametric_heat_conduction -r 0.55 -visit -offline -rdim 16
+//   mpirun -np 8 parametric_heat_conduction -r 0.6 -visit -offline -rdim 16
+//   mpirun -np 8 parametric_heat_conduction -r 0.5 -visit -online -predict

```

Comment:
For predict dmd case, if -visit option is omitted, then the "Elapsed time for predicting DMD" will be so much faster because DMD will only predict the last time step solution.